readme.md

```diff
@@ -50,23 +51,23 @@ The plugin is activated on a per project basis, first you have to add the __Code
 ![CodeChecker Nature Add](docs/nature.png)
 
 #### 3. Configure CodeChecker and checkers
-After that, the settings can be customized in the project preferences window under the CodeChecker panel. For the plugin to work correctly, it is mandatory to add the correct path to the root of the CodeChecker package. You can download and compile CodeChecker from [here](https://github.com/Ericsson/codechecker).
+
+After the __CodeChecker Nature__  is added to the the project, the plugin can be configured globally in `Window -> Preferences -> CodeChecker` panel. Or for the individual project, from the `right click context menu -> Properties -> CodeChecker` page.
+In the first section, you select how and which CodeChecker will be used. After that set some analysis related settings. You should build CodeChecker in a __standalone package__ configuration, and add it to the PATH environment variable, and use the Search in PATH option. Alternatively you can specify a different instance with the Pre built package option. But be aware that the plugin not supports virtual environment, that CodeChecker needs in the default configuration. You can download and compile CodeChecker from [here](https://github.com/Ericsson/codechecker).
```

Comment:
I think an example would be nice here. I did not found any documentation in the codechecker repo about the standalon package mode (maybe I missed it).
```sh
cd codechecker
make standalone_package
export PATH=$PWD/build/CodeChecker/bin:$PATH
```